pages/tutorials/getting_started.md.erb

```diff
@@ -42,8 +42,34 @@ Congratulations, you have run your first Buildkite build! :tada:
 
 To invite your team so they can see your build, go to your organization’s *Settings*, and under *Users* you’ll be able to paste in their email addresses.
 
-## Using a private repository
+## Use a private repository (optional)
 
 When you create a new pipeline with a private repository URL you’ll be shown instructions for configuring your SCM’s webhooks (in GitHub, Bitbucket, etc). Once you’ve followed those instructions make sure your [agent’s SSH keys](/docs/agent/v3/ssh-keys) are configured, and you’ll be good to run a build of your private pipeline.
 
 For more advanced pipelines it can be a good idea to use your development machine as the agent for your first few builds. That way all the dependencies are ready and you’ll sooner be able to share a link to a green build with the rest of your team.
+
+## Enable extra features by upgrading (optional)
```

Comment:
I keep going round and round on this, and I think it feels too much like a sales push here in this doc.

Let's break it out into `pages/enterprise_features.md.erb`

Reply:
We can add a single line call out at the end of the getting started

Reply:
Did as requested, with minor changes:
- The file is `standard_and_enterprise_features.md.erb` as it contains a description of both Standard and Enterprise features.
- Squeezed it into the bottom of Pipelines in the sidebar, right after Audit Log